server/graphql/v2/object/PaymentMethod.js

```diff
@@ -26,6 +29,66 @@ export const PaymentMethod = new GraphQLObjectType({
           return paymentMethod.name;
         },
       },
+      service: {
+        type: GraphQLString,
+        resolve(paymentMethod) {
+          return paymentMethod.service;
+        },
+      },
+      type: {
+        type: GraphQLString,
+        resolve(paymentMethod) {
+          return paymentMethod.type;
+        },
+      },
+      balance: {
+        type: GraphQLInt,
+        description: 'Returns the balance in the currency of this paymentMethod',
+        async resolve(paymentMethod, args, req) {
+          const balance = await paymentMethod.getBalanceForUser(req.remoteUser);
+          return balance.amount;
+        },
+      },
+      currency: {
```

Comment:
Should be a `Currency` type

Reply:
There isn't a `Currency` type right now. If we changed the `balance` to an `Amount` type then there would be a `currency` subfield on that.

Reply:
We do have an Enum type for currency.

But maybe this field is not actually needed, especially if we're moving the balance to an `Amount` type.